htdocs/compta/bank/class/account.class.php

```diff
@@ -1165,7 +1165,7 @@ public function fetch($id, $ref = '')
 				$this->min_allowed    = $obj->min_allowed;
 				$this->min_desired    = $obj->min_desired;
 				$this->comment        = $obj->comment;
-
+				$this->solde		  = $this->solde(1);
```

Comment:
this->solde is not an intrasec property of the object account. It is a calculated field.
So it should not be part of a fetch (a fetch load property of table/extrafields and sub lines if there is).
Subrequest must be called by the caller if the caller need them.

Reply:
oki, so i propose another way